website/src/pages/main.tsx

```diff
@@ -56,7 +56,8 @@ export const Main: React.FC = () => {
         <Typography variant="h1">
           <Link to="/">garn</Link>
         </Typography>
-        <GithubLink />
+        <DiscordLink href={"https://discord.gg/XtDrPsqpVx"} title={"Join the Discord Channel"}/>
+        <GithubLink href="https://github.com/garnix-io/garn" title="View on Github"/>
```

Comment:
We're now passing in the link and the title to the `GithubLink`. That made sense to me, but I can also inline that again, if preferred.